kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -42,27 +44,31 @@ define([
                             [
                                 th(
                                     {
-                                        class: `${cssBaseClass}__table_head_cell--object`,
+                                        class: `${cssBaseClass}__table_head_cell--import-type`,
+                                        style: 'width: 25%',
```

Comment:
I hate having to do this, but it seems the only way* to stop DataTables assigning an absolute width to the table.

* I haven't yet investigated if there is an official way to do it. Going to do that now.

Reply:
Ok. That's fine. Datatables has some fun idiosyncrasies around its styling.

Reply:
You might be able to get around it if you declare all the column headers in the DataTables call.